accepted/PSR-7-http-message.md

```diff
@@ -1697,7 +1704,6 @@ interface UriInterface
      *
      * @param string $query The query string to use with the new instance.
      * @return self A new instance with the specified query string.
-     * @throws \InvalidArgumentException for invalid query strings.
```

Comment:
there is no invalid query string possible and no invalid URI possible by modifying the query.